addons/web/static/tests/core/model_field_selector_tests.js

```diff
@@ -416,4 +416,53 @@ QUnit.module("Components", (hooks) => {
 
         assert.verifySteps(["fields_get"]);
     });
+
+    QUnit.test("Edit path in popover debug input", async (assert) => {
```

Comment:
"debug" input ? Isn't this misleading?

Reply:
the class o_field_selector_debug used it test make it explicit but how would you call it? In a model field selector, there
are only two inputs both in the popover, one used to search and the other the "debug" input